tests/integration/query/commits/branchables/cid_doc_id_test.go

```diff
@@ -49,7 +49,7 @@ func TestQueryCommitsBranchables_WithCidAndDocIDParam(t *testing.T) {
 				Results: map[string]any{
 					"commits": []map[string]any{},
 				},
-				ExpectedError: "cid does not belong to document",
+				// ExpectedError: "cid does not belong to document",
```

Comment:
question(all): To make the commit subscription work properly with `docID` filters, this behavior needed to change. Happy to discuss, but I don't think theres much we can do here. It also feels appropriate to simply not return any results if the `cid` filter doesn't match the `docID` filter, instead of an error.